gamebook/tests/test_nextscene.py

```diff
@@ -2,9 +2,7 @@
 from gamebook.scene import Scene
 from gamebook.tests.test_scene import scene1
 
-nextscene1 = NextScene(scene1, 1.0)
-
-""" scene1: name = "scene1" """
-
 def test_get_scene_name():
+    nextscene1 = NextScene(scene1, 1.0)
```

Comment:
add scene1 to the test for it to be more understandable now.

Reply:
i already merged it :(

Reply:
can i undo it?

Reply:
next PR.

Reply:
:+1: